project/Dependencies.scala

```diff
@@ -5,93 +5,109 @@ object Dependencies {
-    "com.google.guava" % "guava" % "19.0",
-    "com.typesafe.akka" %% "akka-testkit" % akkaV % "test",
-    "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+    sprayJson,
+    sprayHttp,
+    akkaActor,
+    akkaTestkit,
+    scalatest,
+
+    googleCloudBilling,
+    googleGenomics,
+    googleStorage,
+    googleCloudResourceManager,
+    googleCompute,
+    googleAdminDirectory,
+    googlePlus,
+    googleOAuth2,
+    googlePubSub,
+    googleServicemanagement,
+    googleGuava
   )
 
   val utilDependencies = Seq(
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
-    "com.typesafe.akka" %% "akka-actor" % akkaV,
-    "com.typesafe.akka" %% "akka-testkit" % akkaV % "test",
-    "org.scalatest" %% "scalatest" % "2.2.4" % "test",
-    "org.mockito" % "mockito-core" % "2.7.22" % "test"
+    scalaLogging,
+    akkaActor,
+    akkaTestkit,
+    scalatest,
+    mockito
   )
 
   val modelDependencies = Seq(
     // I am not certain why I need jackson-core here but IntelliJ is confused without it and tests don't run
-    "com.fasterxml.jackson.core" % "jackson-core" % "2.4.3",
-    "io.spray" %% "spray-json" % sprayV,
-    "io.spray" %% "spray-http" % sprayV,
-    "io.spray" %% "spray-httpx" % sprayV,
-    "joda-time" % "joda-time" % "2.9.4",
-    "org.joda" % "joda-convert" % "1.8",
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
-    "com.google.api-client" % "google-api-client" % "1.22.0" exclude("com.google.guava", "guava-jdk5"),
-    "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+    jacksonCore,
+    sprayJson,
+    sprayHttp,
+    sprayHttpx,
+    jodaTime,
+    jodaConvert,
+    scalaLogging,
+    googleApiClient,
+    scalatest
   )
 
-  val rawlsCoreDependencies = modelDependencies ++ Seq(
+  val rawlsCoreDependencies = modelDependencies ++ googleDependencies ++ Seq(
     "com.typesafe" % "config" % "1.3.0",
     "com.gettyimages" %% "spray-swagger" % "0.5.0"
       exclude("com.typesafe.scala-logging", "scala-logging-slf4j_2.11")
       exclude("com.typesafe.scala-logging", "scala-logging-api_2.11")
       exclude("com.google.guava", "guava"),
     "com.getsentry.raven" % "raven-logback" % "7.8.6",
-    "com.typesafe.akka" %% "akka-actor" % akkaV,
-    "com.typesafe.slick" %% "slick" % "3.1.1",
-    "io.spray" %% "spray-can" % sprayV,
-    "io.spray" %% "spray-routing" % sprayV,
-    "io.spray" %% "spray-client" % sprayV,
+    slick,
+    slickHikariCP,
+    sprayCan,
+    sprayRouting,
+    sprayClient,
     "org.webjars" % "swagger-ui" % "2.1.1",
     "org.apache.commons" % "commons-jexl" % "2.1.1",
     "org.broadinstitute" %% "wdl4s" % "0.11" exclude("org.scalaz", "scalaz-core_2.11") exclude("io.spray", "spray-json_2.11"),
     "org.scalaz" % "scalaz-core_2.11" % "7.1.3",
     "org.broadinstitute.dsde.vault" %% "vault-common" % "0.1-15-bf74315",
-    "com.google.apis" % "google-api-services-cloudbilling" % "v1-rev7-1.22.0"
```

Comment:
These are all in `googleDependencies`.  `rawlsCoreDependencies` included all of `googleDependencies` already, so I was able to just add the whole thing above.